sky/serve/replica_managers.py

```diff
@@ -673,6 +700,28 @@ def _probe_all_replicas(self) -> None:
                 if info.status_property.first_ready_time is None:
                     info.status_property.first_ready_time = probe_time
             else:
+                if self.use_spot:
```

Comment:
```suggestion
            else:
                handle = global_user_state.get_handle_from_cluster_name(info.cluster_name)
                if handle.launched_resources.use_spot:
```
Shall we read from the `handle.launched_resources` here?

Reply:
yep that should work!